src/strings.ts

```diff
@@ -163,3 +163,10 @@ export const Semver = PatternString(
 );
 
 export type Semver = t.TypeOf<typeof Semver>;
+
+/**
+ * A valid ULID string
+ */
+export const Ulid = PatternString("^[0-9A-HJKMNP-TV-Z]{26}$");
```

Comment:
ULID should be case insensitive

Reply:
fixed in https://github.com/pagopa/io-ts-commons/pull/404/commits/7171960c8ea51acad761cbb9d51010ae4848166f